### PR TITLE
fix: change navbar collapse to offcanvas

### PR DESCRIPTION
**Pull request by dnlggr**
Changes the mobile version of the navbar from a [collapse](https://getbootstrap.com/docs/5.0/components/collapse/) to an [offcanvas](https://getbootstrap.com/docs/5.0/components/offcanvas/) drawer.

It would be nice to show the offcanvas drawer under the navbar somehow but I couldn't find a good way of doing that (except maybe hardcoding a margin but not sure how good that would be).

### CODE REVIEW

**Comment by dennisreimann** [comment]
Updated it, I think it's good now :)

nav

**Review by dennisreimann — APPROVED** [review]

File: src/index.css [hunk 1]
```diff
@@ -202,6 +202,77 @@ body,
...
+  height: var(--line-thickness);
+  top: 50%;
+  left: 50%;
+  background: currentColor;
+  transform: translate(-50%, -50%);
+  transition: transform var(--transition-duration) var(--transition-easing);
+}
+
+#mainNavToggle:hover span {
+  /* color: var(--btcpay-header-text); */
```
Comment (dergigi) [inline src/index.css:261]: Dead code?

**Review by dergigi — APPROVED** [review]
Nice! Looks good to me  👍

File: src/index.css (hunk 1 above)
Comment (dennisreimann) [reply src/index.css:261]: Oh yeah, let's remove it before merging :)

**Comment by dnlggr** [comment]
Cool thanks for the updates—looks great now! 👏

**Pull request merged**